### ISSUE TITLE
Refactor: Expander component

### ISSUE THREAD

**Issue by kaljarv** [OpenVAA/voting-advice-application#341]
The `Expander` component has a few issues, which need refactoring.

### Accessibility

`Aria-label` should be removed, for example.

For the Aria pattern, see: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-faq/

See also the [discussion](https://github.com/OpenVAA/voting-advice-application/pull/316/files#r1435137206) at the PR that introduced the component. Accessibility was also discussed but the comment is lost due to force push.

### Badges

<img width="392" alt="image" src="https://github.com/OpenVAA/voting-advice-application/assets/7520743/72ab7aa5-33d9-417b-add1-755e65a21d68">

In the designs, `Expander`s may have badges. These cannot currently be added, so the title may need to be converted into a slot.

### Edge-to-edge styling

<img width="310" alt="image" src="https://github.com/OpenVAA/voting-advice-application/assets/7520743/2eec00b9-6139-4add-b0e5-26fa8408aeb2">

Currently this edge-to-edge styling cannot be accomplished. To make it work we need to apply `edgetoedge-x` style to the container, title and content or something like that. Merely applying it to the title does not work, because the container has `overflow-x: hidden`.
